src/v/cloud_storage_clients/abs_client.cc

```diff
@@ -541,6 +541,14 @@ ss::future<result<T, error_outcome>> abs_client::send_request(
             vlog(abs_log.debug, "BlobNotFound response received {}", key);
```

Comment:
> cloud_storage_clients: modify abs_client::delete_object()

nit: could you make this more descriptive? Maybe `abs_client: catch not-supported-on-directory errors in delete_object()`. FWIW IMO slightly longer commit message lines are preferable over shorter, less descriptive ones

Reply:
Ah, I've been trying to stick to the `50` character rule. Will prioritize descriptive commit lines from now on.

Reply:
Yea, definitely a balance to strike there, but something concise and still descriptive is super helpful for quickly auditing git history

Reply:
Fixed